src/Components/test/testassets/BasicTestApp/InteropTest/ByteArrayInterop.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.JSInterop;
+
+namespace BasicTestApp.InteropTest
+{
+    public class ByteArrayInterop
+    {
+        [JSInvokable]
+        public static byte[] RoundTripByteArray(byte[] byteArray)
+        {
+            return byteArray;
+        }
+
+        [JSInvokable]
+        public static async Task<byte[]> RoundTripByteArrayAsync(byte[] byteArray)
+        {
+            await Task.Yield();
+            return byteArray;
+        }
```

Comment:
No need for `async` and `Task.Yield` here. `Task.FromResult(byte)` does the trick